frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -215,15 +215,23 @@ export async function action({ context, request }: Route.ActionArgs) {
         ),
       ]) satisfies v.GenericSchema<PrimaryDocumentsSessionData>;
 
+      function toDateString(year: number, month: number, day: number): string {
+        try {
+          return toISODateString(year, month, day);
+        } catch {
+          return '';
+        }
+      }
```

Comment:
I think this should be module-scoped since it doesn't rely on the surrounding closure.